app/assets/javascripts/basetypes/Any.js

```diff
@@ -0,0 +1,17 @@
+const mongoose = require('mongoose');
+const cql = require('cql-execution');
+
+function Any(key, options) {
+  mongoose.SchemaType.call(this, key, options, 'Any');
+}
+Any.prototype = Object.create(mongoose.SchemaType.prototype);
+
+Any.prototype.cast = (any) => {
+  if (any && any.value && any.unit) {
+    return new cql.Quantity(any);
+  }
```

Comment:
What happens if its an Interval, or an Array of things?

Reply:
Depends on how modular we want to make it in the future. The only cases I saw in the ModelInfo file were either types of numbers, a Code, or a Quantity, and only Quantity needed to be casted. At one point, I had checks for Interval or DateTime though, so I could add those back in if you want to future proof it.

Reply:
Yeah let's do that - QDM 6.0 is on the horizon.